docs/source/data/subscriptions.mdx

```diff
@@ -40,20 +40,108 @@ subscription PostFeed {
 
 > Each subscription operation can subscribe to only _one_ field of the `Subscription` type.
 
-## Setting a subscription endpoint
+## Introducing subscriptions to your Apollo Server
 
-Because subscriptions use WebSocket instead of HTTP, Apollo Server uses a _second_ GraphQL endpoint specifically for subscriptions. This endpoint uses the `ws` protocol instead of `http`.
+> Subscriptions are **not** supported by the "batteries-included" `apollo-server` package. In order to implement subscriptions, you must first "eject" (TODO: link to ejecting docs) and use the `apollo-server-express` package (or any other Apollo Server integration package which supports subscriptions). The following steps assume you've already ejected and are using `apollo-server-express`.
```

Comment:
@StephenBarlow I think you are changing to use the term "swap" now right? I know the relevant PRs haven't merged yet but do we know what the link will be?